pkg/commands/loaders/branches.go

```diff
@@ -23,6 +23,8 @@ import (
 // if we find out we need to use one of these functions in the git.go file, we
 // can just pull them out of here and put them there and then call them from in here
 
+const SEPARATION_CHAR = "|"
```

Comment:
Is this still used elsewhere? Also I've been meaning to ask, why have we switched from using this to using the null code?

Reply:
This file used to use `SEPARATION_CHAR`, but now uses the null code.

Reply:
Looking at the file on master, it's still using `SEPARATION_CHAR` so my question is: why are we opting for the null code instead of the original separation character? does using '|' cause issues in some cases e.g. when the output itself contains '|'?

Reply:
@Ryooooooga in case you missed my comment ^ I'm happy to merge the code already but just wanted clarification on that

Reply:
@jesseduffield branch names can contain `|`.
master:
<img width="537" alt="master" src="https://user-images.githubusercontent.com/10097437/160504795-5b337a1a-5086-4c57-9891-c17941a75802.png">
PR:
<img width="540" alt="PR" src="https://user-images.githubusercontent.com/10097437/160504797-44ac8c10-7011-4606-be6a-9d34a7f01a3e.png">

Reply:
ah cool, very well!